packages/lib/src/vis/shared/SelectionMesh.tsx

```diff
@@ -14,48 +16,73 @@ interface Props {
 
 function SelectionMesh(props: Props) {
   const { selectionComponent: Selection } = props;
+  const { abscissaScale, ordinateScale } = useAxisSystemContext();
+
+  const worldToData = useCallback(
+    (vec: Vector3) =>
+      new Vector2(abscissaScale.invert(vec.x), ordinateScale.invert(vec.y)),
+    [abscissaScale, ordinateScale]
+  );
+
+  const dataToWorld = useCallback(
+    (vec: Vector2) => new Vector2(abscissaScale(vec.x), ordinateScale(vec.y)),
+    [abscissaScale, ordinateScale]
+  );
```

Comment:
These are very nice. Could you maybe move them to the axis system context so we can re-use them in other places? They'll be very convenient to use in the new `useVisibleDomain` hook I was about to work on...

Reply:
Sounds good.

Should we keep `Vector2` or use a more generic structure like `[number, number]` ?

Reply:
`Vector2` is great actually!

Reply:
it still represent a point in a different space.